src/tests/details.test.js

```diff
@@ -0,0 +1,14 @@
+import renderer from 'react-test-renderer';
+import { BrowserRouter as Router } from 'react-router-dom';
+import Details from '../componenets/details/Details';
+
+it('renders correctly', () => {
+  const tree = renderer
+    .create(
+      <Router>
+        <Details />
+      </Router>,
+    )
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] You did an excellent job adding the snapshot for the **Details component** :clap: . However, after running the test, you will receive a **warning**. It's because we're passing some props to the ***Details component** and using **props type** to validate those props and mark them as required. So, when testing the component, we should pass some props as examples to ensure that the test passes without warning. 
#### This :point_down:  is the warning
![Screenshot from 2023-05-05 23-40-17](https://user-images.githubusercontent.com/89228743/236586411-a133f458-5f74-43e2-a5f0-cd9c91433dac.png)


#### To help you, I would suggest you update the test case like this :point_down: 

```suggestion
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Details from '../componenets/details/Details';

it('renders correctly', () => {
  const props = {
    population: 100000,
    area: 2000,
    flags: 'https://example.com/flag.png',
    official: 'Official Name',
    name: 'Country Name',
    unMember: 'Yes',
    languages: 'Language',
    currencies: 'Currency',
    independent: 'Yes',
    region: 'Region',
  };
  const tree = renderer
    .create(
      <Router>
        <Details {...props} />
      </Router>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
```


## Important !
After you've added those suggestions, make sure to type the letter **u** on your keyboard after running `npm test` to update the snapshots. Like this :point_down:  Then, to see all of the tests, type **a** on your keyboard.
 
![Screenshot from 2023-05-06 00-13-40](https://user-images.githubusercontent.com/89228743/236587617-cb1cb463-3d13-43f7-8854-ed5d27511c97.png)

Reply:
Thanks for Your Suggestion. 👍 👍 
 The warning is fixed now.